test-data/fake-virtio-win/Makefile.am

```diff
@@ -31,16 +39,14 @@ CLEANFILES += drivers cd fake-virtio-win.iso
 
 drivers: virtio-win-drivers-list.txt
 	while read line; do			\
-		path="$@/$$line";		\
-		mkdir -p `dirname $$path`;	\
-		touch "$$path";			\
+		mkdir -p `dirname $$line`;	\
+		touch "$$line";			\
 	done < $<
```

Comment:
Does this hunk (and the one below) change the path that is created?  I guess it should be in a separate commit or removed if added unintentionally.

Reply:
Actually it's a bug fix for the first patch, but I squashed it incorrectly. That patch was just pushed though. Keep it as is?